src/Go/boardState/boardState.ts

```diff
@@ -122,6 +123,7 @@ export function makeMove(boardState: BoardState, x: number, y: number, player: G
 
   // Add move to board history
   boardState.previousBoards.unshift(boardStringFromBoard(boardState.board));
+  clearAllPointHighlights();
```

Comment:
This isn't in the right place. `makeMove` can be applied to any `BoardState`, whereas highlights are scoped to the global go object.

Either that, or really highlights should be part of BoardState and just not saved with it. I could see either way, but the object model right now doesn't work right.

Reply:
I moved to this to be a property of the boardstate. It makes more sense there.

Reply:
You've still got the same issue. `clearAllPointHighlights` is a method on NSGoImpl, which is hardcoded to use Go.currentGame.boardState. Functions within `BoardState` need to modify their *own* BoardState.

Reply:
I moved this to be part of the Go object, and they interact with their own this.currentGame object. Is that what you were thinking?